apps/examples/log_dump/log_dump_main.c

```diff
@@ -43,6 +43,7 @@ int log_dump_main(int argc, char *argv[])
 #endif
 {
 	int ret = 1;
+	int total_read = 0;
```

Comment:
Sorry but, I want to keep separate commit for main change and app change. Could you split a commit for this app? It will be easy to sync later.